### PR TITLE
Update Version.h, Version.cpp with new constant notation part 1 #21421

### PR DESCRIPTION
**Pull request by oribro**

### CODE REVIEW

File: src/openrct2/command_line/RootCommands.cpp [hunk 1]
```diff
@@ -71,9 +71,9 @@ static constexpr CommandLineOptionDefinition StandardOptions[]
     { CMDLINE_TYPE_SWITCH,  &_version,          'v', "version",            "show version information and exit"                          },
     { CMDLINE_TYPE_SWITCH,  &_noInstall,        'n', "no-install",         "do not install scenario if passed"                          },
     { CMDLINE_TYPE_SWITCH,  &_all,              'a', "all",                "show help for all commands"                                 },
-    { CMDLINE_TYPE_SWITCH,  &_about,            NAC, "about",              "show information about " OPENRCT2_NAME                      },
+    { CMDLINE_TYPE_SWITCH,  &_about,            NAC, "about",              "show information about OpenRCT2"                      },
```
Comment (Gymnasiast) [inline src/openrct2/command_line/RootCommands.cpp:74]: Please realign the `}` (on both lines you changed).

**Comment by Gymnasiast** [comment]
Do not address my comment(s) yet, there is more to come.

**Comment by ZehMatt** [comment]
I've looked at the changes and I think this PR is touching on too much legacy code so I'm not sure if we shouldn't rather clean it all up instead on trying to work around it. All the passing of `c_str()` is quite bad, I also don't like how we store the version data, ideally we just have something like `static constexpr VersionInfo _versionData = { major, minor, ... }` and construct strings at runtime for when the version is actually needed as a string, Version.h could simply have a few functions that provide this such as `const VersionInfo& getVersionInfo();` `std::string getVersionString();` and so on. So I don't think we should merge this.

**Review by ZehMatt — COMMENTED** [review]
I added my remarks to what should be improved, this is beyond just changing the code style so that should be addressed in other PR(s).

**Comment by Gymnasiast** [comment]
@oribro Please let us know if you want to pick up ZehMatt’s suggestions and we can work out how to tackle it. (If you don’t want to work on it, that’s fine too, but please let us know.)

**Comment by Gymnasiast** [comment]
Closing this as there has been no answer and the PR still needs a lot of work.

**Pull request closed without merging**